packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [38.2.0]
+
+### Changed
+
+- Fetch token rates for all accounts instead of only the selected account ([#4759](https://github.com/MetaMask/core/pull/4759))
```

Comment:
It's also worth noting that this change specifically applies to the `TokenRatesController`.
```suggestion
- The TokenRatesController now fetches token rates for all accounts, instead of just the selected account ([#4759](https://github.com/MetaMask/core/pull/4759))
```